databases/gke-stateful-postgres/terraform/modules/beta-autopilot-private-cluster/versions.tf

```diff
@@ -21,7 +21,7 @@ terraform {
   required_providers {
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = "< 5.31"
+      version = "< 5.32"
```

Comment:
Thought: I wonder if there was a particular reason for enforcing this constraint.

Reply:
My quick guess: In an environment with tests running on code changes, this allows you to assert that the version you're pinned to will work (since the tests are passing). If you remove the requirement, then the underlying version may change over time, but since there would be no code changes (like this PR) then we would never re-run the tests validating the the new version still works.